src/Helpers/SshCommandTrait.php

```diff
@@ -66,4 +67,14 @@ protected function findLocalSshKeys(): array {
     return iterator_to_array($finder);
   }
 
+  /**
+   * @param \Symfony\Component\Console\Style\SymfonyStyle $io
+   *
+   * @return bool
+   */
+  protected function promptWaitForSsh(SymfonyStyle $io): bool {
+    $io->note("It may take an hour or more before the SSH key is installed on all of your application's servers. Create a Support ticket if this process takes too long.");
```

Comment:
Maybe add something like "Please try again if it times out or create a Support ticket to get more assistance."